hawc/apps/assessment/migrations/0030_assessment_models.py

```diff
@@ -0,0 +1,96 @@
+import django.db.models.deletion
```

Comment:
can these 4 migrations be changed into one? It might have to be manual, migration, not a squash. It may not be possible, I know there's some magic here, but if that's the case, just have two migrations, one for the field creates and run python operations, and one for the old column removals. you could name them `_assessment_models_part_1` and part 2, etc.